New version: xushengfeng.eSearch version 14.6.0

xushengfeng opened the pull request:
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.11.0 :rocket:

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/239174)

xushengfeng committed:
New version: xushengfeng.eSearch version 14.6.0

stephengillie commented:
Hi @xushengfeng,

The `InstallerUrl` seems to have changed, and the `PackageUrl` no longer leads to it. Is there another page that has a link to this new `InstallerUrl`?

xushengfeng committed:
remove dev install url

The previous url was the url of GitHub accelerated mirror. I forgot to remove it. Since it is a mirror, the file content is the same. For common use, the mirror that is removed now
